src/AspNet.Security.OpenIdConnect.Extensions/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.0-beta5-*",
+  "version": "1.0.0-beta6-*",
```

Comment:
Hum, looks like you based your branch on the `dev` branch instead of the `release` branch.
Don't worry, I'll fix that when merging it :smile: